### ISSUE TITLE
Game default music is still playing on 0.18

### ISSUE THREAD

**Issue by langaster** [jocker-il/JKIL-KnightRider#8]
Default game music does not mute when entering a vehicle on 0.19.4.

**Comment by jocker-il** [comment]
Yes,  the music this mod adds is instead of the car engine, not the game music,  so it makes sense it doesn't stop - I play with game music on low setting so I didn't notice this issue. 

 But changing the game music volume should be agreed by all for me to even research shutting it down

so we need a poll - get one going and come back to me with results
